WORKSPACE

```
@@ -466,7 +465,7 @@ load("@aspect_rules_swc//swc:repositories.bzl", "swc_register_toolchains")
 
 swc_register_toolchains(
     name = "swc",
-    swc_version = "v1.2.141",
+    swc_version = "v1.3.78",  # LATEST_SWC_VERSION
```

nit: this comment could easily go out of date

```suggestion
    swc_version = "v1.3.78",
```

---

I was thinking that it would help urge whoever upgrading our rules_swc next to also upgrade the swc_version 🌝 

But that's probably not needed.